.github/workflows/crawler.yml

```diff
@@ -3,14 +3,14 @@
 name: Crawler
 
 on:
-  # push:
-  #   branches: [ master, main ]
-  #   tags: [ v* ]
-  # pull_request:
-  # schedule:
-  # - cron: '0 14 * * *'
+  push:
+    branches: [ master, main ]
+    tags: [ v* ]
+  pull_request:
+  schedule:
+  - cron: '0 14 * * *'
```

Comment:
@andrew2net maybe we should keep this PR till https://github.com/relaton/support/issues/6 not resolved? Because `keep-alive` doesn't make much sense in this case

Reply:
I agree

Reply:
Fixed on repo level, @andrew2net could you please review again